tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4

```diff
@@ -0,0 +1,196 @@
+#
+# Topology for Icelake with rt711 + rt1308 (x2) + rt715.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include ICL DSP configuration
+include(`platform/intel/icl.m4')
+
+DEBUG_START
+
+#
+# Define the pipelines
+#
+# PCM0 ---> volume ----> ALH 2 BE dailink 0
+# PCM1 <--- volume <---- ALH 3 BE dailink 1
+# PCM2 ---> volume ----> ALH 2 BE dailink 2
+# PCM3 ---> volume ----> ALH 2 BE dailink 3
+# PCM4 <--- volume <---- ALH 2 BE dailink 4
+# PCM5 ---> volume <---- iDisp1
+# PCM6 ---> volume <---- iDisp2
+# PCM7 ---> volume <---- iDisp3
+
+dnl PIPELINE_PCM_ADD(pipeline,
+dnl     pipe id, pcm, max channels, format,
+dnl     period, priority, core,
+dnl     pcm_min_rate, pcm_max_rate, pipeline_rate,
+dnl     time_domain, sched_comp)
+
+# Low Latency playback pipeline 1 on PCM 0 using max 2 channels of s32le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+	1, 0, 2, s32le,
```

Comment:
sorry, one question, do we really need 32bit stream for playback & capture ?
16 or 24 bit is more reasonable and saves bandwidth.

Reply:
@RanderWang 32bits is internal to give headroom for 24bit data processing.

Reply:
@lgirdwood  sorry, I remarked at wrong point, which should be dai setting. 32bit dai will affect SDW bus setting and codec setting. Please check our dai fix function, which will force codec and SDW bus with the same format of dai. 

And 16bit stream costs less memory.  As a audiophile, I can't tell the difference with 16bit or 24bit stream on Hi-End system valued thousands of dollars, how about cheap devices?

Reply:
@RanderWang ok, so will 32bit cause an issues for SDW ? I think it should be fine.  If not, we can down convert to 16/24 with a volume control prior to the DAI component. @plbossart any comment here ?

Reply:
We use 32 bits everywhere for now, and it helps stress-test the bandwidth allocation algorithm since at the moment we don't have multiple devices per link.
I agree 32 bits is overkill but we will have to revisit all these topologies when we work on synchronized playback so let's use 32 bits for now across the board, except for CML where 24 bits is required due to the bandwidth allocation restrictions.